style.css

```diff
@@ -22,9 +23,13 @@ img {
 
 .header {
   order: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
 }
 
-.toolbar > nav {
+.mobile-toolbar > nav {
```

Comment:
- [ ] Kindly make sure you use the appropriate background image for the main section as seen in the figma design 🙏 
- [ ] Good job 👍. Kindly make sure the content of your main section are given the appropriate margin and padding to align your content as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/186785801-8926b788-84e2-4b04-8251-d405f7da5f5e.png)  |  ![image](https://user-images.githubusercontent.com/74814780/186785673-847d7f98-34fb-4e45-9d73-185371848474.png)

Reply:
Done!